src/account/UpgradeableModularAccount.sol

```diff
@@ -298,14 +296,35 @@ contract UpgradeableModularAccount is
         AccountStorage storage _storage = getAccountStorage();
 
         ModuleEntity sigValidation = ModuleEntity.wrap(bytes24(signature));
+        signature = signature[24:];
 
-        (address module, uint32 entityId) = sigValidation.unpack();
         if (!_storage.validationData[sigValidation].isSignatureValidation) {
-            revert SignatureValidationInvalid(module, entityId);
+            (address _module, uint32 _entityId) = sigValidation.unpack();
```

Comment:
At some point would love to revisit all of our variable naming according to our style guide.

Reply:
Yep, we should pick a standardized way to name vars when avoiding shadowing. RIght now, this pattern is ambiguous with internal-visibility variables and constants.